src/main/kotlin/Shovel.kt

```diff
@@ -24,14 +25,14 @@ fun shovelAsync(consumer: KafkaConsumer<ByteArray, ByteArray>, hbase: HbaseClien
                 )
                 consumer.subscribe(Config.Kafka.topicRegex)
 
-
                 logger.info(
                     "Polling",
                     "poll_timeout", pollTimeout.toString(),
                     "topic_regex", Config.Kafka.topicRegex.pattern()
                 )
 
                 val records = consumer.poll(pollTimeout)
+                //consumer.pause(consumer.assignment())
```

Comment:
What is the plan with this and the resume? We should uncomment or take out really.

Reply:
oops yeah